File: pueblo_golf_tournament_mobile/lib/dto/model/division-dto.dart
import 'package:pueblo_golf_tournament_mobile/base/base-dto.dart';

import 'package:json_annotation/json_annotation.dart';

part 'division-dto.g.dart';

@JsonSerializable()
class DivisionDto extends BaseDto {
  final String name;
  final int tournamentId;
  DivisionDto({
    required super.id,
    required this.name,
    required this.tournamentId,
  });
  factory DivisionDto.fromJson(Map<String, dynamic> json) =>
      _$DivisionDtoFromJson(json);
  Map<String, dynamic> toJson() => _$DivisionDtoToJson(this);
}

File: pueblo-golf-tournament-api/Dto/DivisionDto.cs
using pueblo_golf_tournament_api.Base;

namespace pueblo_golf_tournament_api.Dto
{
    public class DivisionDto : BaseDto
    {
        public string? Name { get; set; }
        public long TournamentId { get; set; }
    }
    public class DivisionRawDto
    {
        public string? Name { get; set; }
        public long TournamentId { get; set; }
    }
}